packages/waku/src/router/client.ts

```diff
@@ -31,6 +31,7 @@ import {
   LOCATION_ID,
 } from './common.js';
 import type { RouteProps, ShouldSkip } from './common.js';
+import type { InferredPaths } from './create-pages/types-from-route-config.js';
```

Comment:
This shouldn't come from `create-pages`. Can you move the definition `InferredPaths` here?